### ISSUE TITLE
Create the electron windows 32-bit download/install

### ISSUE THREAD

**Issue by tangollama** [HospitalRun/hospitalrun-frontend#1074]
**Expected behavior:**
Availability of the code as a win32 download.

**Actual behavior:**
Currently unavailable.

**Steps to reproduce:**
- Checkout the project on a win32 development device.
> npm install
> bower install
> ember electron:make --platform=win32

Talk to me on Slack @tangollama to become part of the build team process.

**Comment by andrew-werdna** [comment]
![unsupported_platform](https://cloud.githubusercontent.com/assets/8261769/26339336/e6b52f10-3f4c-11e7-9876-b9d9f3f08ea5.png)

^ not to raise any alarms here @tangollama , ^ I just saw this particular task. For the task we just closed out, #1075 , when I used the command specified in *that* task, it gave me this error. So for #1075 I used the command featured in *this* task. It seemed to still generate the windows 64-bit executable unless I'm super mistaken here (My laptop is 64-bit OS and the executable worked on my machine).

Again, not trying to alarm you, but is there a different command for 32-bit vs 64-bit Windows OS? To summarize I used the command `ember electron:make --platform=win32` on #1075 because when I ran the command `ember electron:make --platform=win32x64` I got the unsupported platform error in the attached screenshot. I assumed that the **Steps To Reproduce** from #1075 just had an error/type-o in the command. I'll look into this tomorrow as well, but if you're willing/able, let's follow up on this and make sure we all have the same understanding.

**Comment by tangollama** [comment]
My understanding is that the install for win32 needs to be executed on a win32 OS. electron forge (the underlying build project) is opinionated regarding building for other architectures. 

Typed or dictated imperfectly on my iPhone (717.385.9970)

**Comment by andrew-werdna** [comment]
Ah, understood!

**Comment by baoqchau** [comment]
Hi @tangollama, I tried to run the Steps to reproduce as you said above but unfortunately I encountered a bug: 
![2017-06-27 21_42_29-c__windows_system32_cmd exe](https://user-images.githubusercontent.com/8473870/27620893-b28f3b62-5b81-11e7-9933-34ecbd326193.png)
I tried to google it and it turned out a bug in electron-forge, which is in this PR here: [ https://github.com/electron-userland/electron-forge/pull/161](url). They nearly finish the PR, according to their conversation. So I think that I will keep track of that PR and try to come back here after they merge that PR.
